sql/maintenance_utils.sql

```diff
@@ -39,7 +39,7 @@ CREATE OR REPLACE FUNCTION @extschema@.compress_chunk(
 
 CREATE OR REPLACE FUNCTION @extschema@.decompress_chunk(
     uncompressed_chunk REGCLASS,
-    if_compressed BOOLEAN = false
```

Comment:
pgspot complains about the redefinition of this function. You need to add this change to the upgrade/downgrade scripts as well.